Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get update -qy && apt-get dist-upgrade -qy && apt-get install -o Dpkg::O
     libtiff-dev libjpeg-dev libopenjp2-7-dev libwebp-dev zlib1g-dev \
     graphviz \
     locales \
-    nodejs && apt-get -qy -t stretch-backports install npm && apt-get -qy autoremove && apt-get -qy autoclean
+    nodejs node-gyp && apt-get -qy -t stretch-backports install npm && apt-get -qy autoremove && apt-get -qy autoclean
```

Comment:
The `node-gyp` package might not be installable from NPM just due to upstream breakage, which is common enough that I remember it by name. Installing it from APT should be a big stability improvement.